### ISSUE TITLE
Test count in runner keeps increasing during run

### ISSUE THREAD

**Issue by martijnburgers** [ipjohnson/SeleniumFixture#8]
Because a single test can run for various browsers the amount of tests to be run is currently unknown or just wrong at the beginning of a test-run and keeps increasing during the run.

It's not really a big issue but it would be nice if the amount of tests is calculated once at the beginning of a test-run. 

I can imagine that this is impossible or hard to implement.

**Comment by ipjohnson** [comment]
Yeah that's more a function of the way xUnit discovery works. Originally when I was implementing I really didn't see a way around it. That said it's worth taking a look at some point to see if this has changed.
